common/src/main/scala/explore/model/AppContext.scala

```diff
@@ -29,19 +33,29 @@ case class AppContext[F[_]](
   pageUrl:       (AppTab, Program.Id, Focused) => String,
   setPageVia:    (AppTab, Program.Id, Focused, SetRouteVia) => Callback,
   environment:   ExecutionEnvironment
-)(implicit
+)(using
   val F:         Applicative[F],
   val logger:    Logger[F],
   val P:         Parallel[F]
-) {
+):
   def pushPage(appTab: AppTab, programId: Program.Id, focused: Focused): Callback =
     setPageVia(appTab, programId, focused, SetRouteVia.HistoryPush)
 
   def replacePage(appTab: AppTab, programId: Program.Id, focused: Focused): Callback =
     setPageVia(appTab, programId, focused, SetRouteVia.HistoryReplace)
-}
 
-object AppContext {
+  given WebSocketClient[F, ObservationDB]     = clients.odb
+  given WebSocketClient[F, UserPreferencesDB] = clients.preferencesDB
+  given TransactionalClient[F, ITC]           = clients.itc
+
+  given itcWorker: WorkerClient[F, ItcMessage.Request]         = workerClients.itc
+  given catalogWorker: WorkerClient[F, CatalogMessage.Request] = workerClients.catalog
+  given agsWorker: WorkerClient[F, AgsMessage.Request]         = workerClients.ags
+  given plotWorker: WorkerClient[F, PlotMessage.Request]       = workerClients.plot
+
+object AppContext:
+  val ctx: Context[AppContext[IO]] = React.createContext(null) // No default value
```

Comment:
It's set in scope here: https://github.com/gemini-hlsw/explore/blob/contexts/explore/src/main/scala/explore/RootComponent.scala#L33

The default value (`null` in this case) means that if you attempt `.useContext` from a component that is not embedded within a `provide`, then it will get the default value.

If we wanted, we could run the whole app with another `AppContext` (say, a testing one) just by providing it instead of our current one.